PROV/prov-bfo-directmappings.ttl

```diff
@@ -54,36 +54,47 @@ prov:Agent                rdfs:comment "A prov:Agent is a material entity becaus
-                          rdfs:subClassOf obo:BFO_0000029 . # site
+                          owl:equivalentClass obo:BFO_0000029 . # site
 
 
 prov:InstantaneousEvent   rdfs:comment "An prov:InstantaneousEvent is equivalent to a process boundary because it is referred to as the boundary (start or end) of a prov:Activity as an occurrent."@en ;
                           owl:equivalentClass obo:BFO_0000035 . # process boundary
 
 
-prov:Plan                 rdfs:comment "prov:Plans exists in virtue of the fact that there is at least one of what may be multiple copies; it is the content or the pattern that the multiple copies share."@en ;
+prov:Plan                 rdfs:comment "A prov:Plan exists in virtue of the fact that there is at least one of what may be multiple copies; it is the content or the pattern that the multiple copies share."@en ;
                           rdfs:subClassOf obo:BFO_0000031 . # generically dependent continuant
 
 
-prov:Influence            owl:disjointUnionOf ( obo:BFO_0000015  # process, e.g. prov:Derivation
+prov:Role                 rdfs:comment "A prov:Role is a bfo:role because its externally determined by the context in which the bearer plays the role."@en ;
+                          rdfs:subClassOf obo:BFO_0000023 . # role TODO: realized in an Activity? need to resolve issue with prov:hadRole
+
+
+prov:Influence            rdfs:comment "A prov:Influence is either a process or process boundary, but not both at once, because it occurs over time or at an instant in time. Although prov:Influence is defined as a capacity, all of its subclasses are clearly defined as occurring over time or at an instant."@en ;
+                          owl:disjointUnionOf ( obo:BFO_0000015  # process, e.g. prov:Derivation
                                                 obo:BFO_0000035  # process boundary, e.g. prov:Generation is a prov:InstantaneousEvent
-                                             ) ;
-                          rdfs:comment "prov:Influences are either processes or process boundaries, but not both at once, because they occur over time or at an instant in time. Although prov:Influence is defined as a capacity, all of its subclasses are clearly defined as occurring over time or at an instant."@en .
+                                            ) .
 
 
 # Object Properties
-prov:hadMember            rdfs:subPropertyOf obo:BFO_0000051 . # has part
-prov:hadGeneration        rdfs:subPropertyOf obo:BFO_0000121 . # has temporal part
-prov:hadActivity          rdfs:subPropertyOf obo:BFO_0000139 . # temporal part of
-prov:qualifiedStart       rdfs:subPropertyOf obo:BFO_0000121 . # has temporal part
-prov:qualifiedEnd         rdfs:subPropertyOf obo:BFO_0000121 . # has temporal part
-prov:qualifiedUsage       rdfs:subPropertyOf obo:BFO_0000121 . # has temporal part
\ No newline at end of file
+prov:hadMember            rdfs:comment "A prov:Collection that has member some prov:Entity also has it as a part."@en ;
```

Comment:
In BFO2020, there is indeed "has member part at some time"

Reply:
RO "has member" is a subproperty of BFO "has part" so I was thinking this could also be a subproperty. I agree that we can try to make these into equivalence relations but that requires more axioms. Please suggest any you can think of here or in #11

Reply:
Ok I'll change to subproperty of "has member part at some time"

Reply:
Nevermind that's only between material entities .......... so I will go with "has continuant part"